addon/ng2/utilities/find-parent-module.ts

```diff
@@ -0,0 +1,34 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+module.exports = function findParentModule(
+  project: any,
+  currentDir: string): string {
+
+  const sourceRoot = path.join(project.root, project.ngConfig.apps[0].root, 'app');
+
+  // trim currentDir
+  currentDir = currentDir.replace(path.join(project.ngConfig.apps[0].root, 'app'), '');
+
+  let parts = currentDir.split(path.sep);
+  const partsLength = parts.length;
+  let pathToCheck = path.join(sourceRoot, currentDir);
+
+
+  for (let i = 0; i <= partsLength; i++) {
```

Comment:
Oh I see. Why not do `while (!path.relative(sourceRoot, currentDir)) {` ?  `path.relative` will return empty string if the two paths are the same